kalite/distributed/templates/admin/superuser_form.html

```diff
@@ -8,18 +8,27 @@
         {% endif %}
         <form class="form-horizontal" id="superusercreate-box" method="post">
             <div class="field">
+            <!--[if IE]>
+                <label for="id_superusername" style="float:left">{% trans 'Username' %}</label>
+            <![endif]-->
                 {{ form.superusername.errors }}
                 {{ form.superusername }}
             </div>
             </br>
             <div class="field">
+            <!--[if IE]>
```

Comment:
all other browsers support placeholder in input tag, only IE deserves the ugly labels.